frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/HotplugFieldLevelHelp.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { Trans, useTranslation } from 'react-i18next';
+import { FieldLevelHelp } from '@console/internal/components/utils';
+
+export const HotplugFieldLevelHelp = () => {
+  const { t } = useTranslation();
+  return (
+    <FieldLevelHelp popoverHasAutoWidth>
+      <Trans t={t} ns="kubevirt-plugin">
```

Comment:
this is not a good use for `trans`, it's just two strings that need translation, the extra context you give the translator by giving him/her the title+text  in the same translation string does is not enough to justify adding the extra components, that makes translation harder.

Reply:
applied